.github/workflows/test.yml

```diff
@@ -84,13 +84,24 @@ jobs:
           limactl shell vm1 ip a
       - name: "Lima: vm1: set up iperf3"
         run: |
-          limactl shell vm1 sudo DEBIAN_FRONTEND=noninteractive apt-get install -y iperf3
+          # net-tools for netstat
+          limactl shell vm1 sudo DEBIAN_FRONTEND=noninteractive apt-get install -y iperf3 net-tools
           limactl shell vm1 systemd-run --user --unit=iperf3 iperf3 -s
```

Comment:
why not use the iperf3 service?

    limactl shell sudo systemctl enable iperf3 --now